docs/content/concepts/batches.md

```diff
@@ -10,7 +10,7 @@ significant benefits in terms of storage and compute.
 Some examples of batched data include points in a point cloud, bounding boxes for detected objects, tracked keypoints
 in a skeleton, or a collection of line strips.
 
-In the Python APIs, the majority of archetypes are named with the plural form, for example [`rr.Points3D`](https://ref.rerun.io/docs/python/stable/common/archetypes/#rerun.archetypes.Points3D). They accept both single elements (internally treated as an N=1 batch) or arrays corresponding to the batches.
+In the logging APIs, the majority of archetypes are named with the plural form, for example [`rr.Points3D`](https://ref.rerun.io/docs/python/stable/common/archetypes/#rerun.archetypes.Points3D). They accept both single elements (internally treated as an N=1 batch) or arrays corresponding to the batches.
```

Comment:
are they even logging apis now? I'm not sure what's the collective for "send + log", is it `data ingestion APIs`?

Reply:
I dunno, but we use that term in a bunch of places so let's stick with it for now